r1cs/gadgets/crypto/src/merkle_tree/field_based_mht/mod.rs

```diff
@@ -498,6 +510,17 @@ mod test {
             )
             .unwrap();
 
+            // Test get_leaf_index()
+            let leaf_index_g = cw.get_leaf_index(cs.ns(|| format!("get leaf index {}", i))).unwrap();
+
+            fe_index_g.enforce_equal(cs.ns(|| format!("enforce_leaf_index_{} by calling get_leaf_index() [part 1 of 2]", i)), &leaf_index_g).unwrap();
+
+            cw.enforce_leaf_index(
+                &mut cs.ns(|| format!("enforce_leaf_index_{} by calling get_leaf_index() [part 2 of 2]", i)),
+                &leaf_index_g,
+            )
+            .unwrap();
```

Comment:
We can remove this test case as we enforced equality among `leaf_index_g` and `fe_index_g`, and we have already tested `enforced_leaf_index` with `fe_index_g`

Reply:
Done with commit [ed135ab54ba091b29d8ae7201787772eb54fc269](https://github.com/HorizenOfficial/ginger-lib/pull/165/commits/ed135ab54ba091b29d8ae7201787772eb54fc269).